### DISCUSSION TITLE
I have big problem

### DISCUSSION THREAD

**Discussion by Aleksandra0056** [Aris-t2/CustomCSSforFx#594] [General]
I wanted to add some custom CSS hacks to my Firefox, but they won't load. I cleared startup cache, I added import to userChrome.css and nothing.

Here are my userChrome.css imports:
```
@import "./css/aboutaddons/addonlists_colorized_button_icons.css";
@import "./css/aboutaddons/addonlists_compact_more_compact.css";
@import "./css/aboutaddons/availableupdates_category_always_visible.css";
@import "./css/aboutaddons/recentupdates_category_always_visible.css";
@import "./css/aboutaddons/recommendations_category_always_hidden.css";
```

**Comment by Acid-Crash** [comment]
Hi, 
Seems like you are trying to alter the content of add-on page.
Such changes should be within userContent.css OR in dedicated personal my_userContent.css which should be linked to the primary  userContent.css

**Reply by Aleksandra0056** [reply]
Thank you for your advice, I'm closing this.
